openmetadata-spec/src/main/resources/json/schema/security/sasl/saslClientConfig.json

```diff
@@ -0,0 +1,35 @@
+{
+  "$id": "https://open-metadata.org/schema/security/sasl/SASLClientConfig.json",
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "SASL Client Config",
+  "description": "SASL client configuration.",
+  "type": "object",
+  "javaType": "org.openmetadata.schema.security.sasl.SASLClientConfig",
+  "additionalProperties": false,
+  "properties": {
+    "saslMechanism": {
+      "title": "SASL Mechanism",
+      "description": "SASL security mechanism",
+      "default": "PLAIN",
+      "type": "string",
+      "enum": [
+        "PLAIN"
+      ],
+      "javaEnums": [
+        {
+          "name": "PLAIN"
+        }
+      ]
+    },
```

Comment:
do you think we can refer to sasl mechanism type from here?

https://github.com/open-metadata/OpenMetadata/blob/main/openmetadata-spec/src/main/resources/json/schema/entity/services/connections/messaging/saslMechanismType.json

Reply:
Sure, I've made the change here: 
8d462f0